applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -246,6 +249,7 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestConstitutiveLaw]))
     nightSuite.addTest(TSimpleSmallDeformationPlasticityMCTest('test_execution'))
     smallSuite.addTest(TSimpleSmallDeformationPlasticityVMTest('test_execution'))
+    smallSuite.addTest(TSerialParallelRuleOfMixturesCubeDamageTest('test_execution'))
```

Comment:
how long does this test need? This is the small suite, it should not take more than ~0.05s max

Reply:
It is exactly the same as the previous ones (TSimpleSmallDeformationPlasticityVMTest) but changing the CL to this new composite rule of mixtures :)

Reply:
and how long does that one run? :)

Reply:
let me check

Reply:
Maybe we should move some of the tests to Nightly, including the old ones.

Reply:
In another PR then right?

Reply:
I think it takes like 0.5s so maybe it's too long

Reply:
yep that shoudl go to the night suite